Extension/src/LanguageServer/ui.ts

```diff
@@ -344,6 +344,34 @@ export class UI {
         return (selection) ? selection.index : -1;
     }
 
+    public async showSelectCompiler(paths: string[]): Promise<number> {
+        const options: vscode.QuickPickOptions = {};
+        options.placeHolder = localize("select.compile.commands", "Select a compiler to configure for IntelliSense");
+
+        const items: IndexableQuickPickItem[] = [];
+        for (let i: number = 0; i < paths.length; i++) {
+            let option: string | undefined;
+            let isCompiler: boolean = false;
+            if (paths[i].indexOf("\\") > 0) {
+                if (paths[i].split("\\").pop() !== undefined) {
+                    option = paths[i].split("\\").pop();
+                    isCompiler = true;
+                }
+            } else {
+                isCompiler = false;
+            }
+            if (option !== undefined && isCompiler) {
+                const path: string | undefined = paths[i].replace(option, "");
+                items.push({ label: option, description: localize("found.string", "Found at") + path, index: i });
```

Comment:
Should this dropdown quickpick show a "trusted" or "untrusted" label? Otherwise, I'm worried users might assume all the compilers in the list are "trustable" when they may not be, i.e. showing "untrusted" may emphasize that choosing it is an indication of their trusting it.

Reply:
Can/should a "current" label be added for the current default? That would make it easier to see what I'm switching away from using.

Reply:
For now, there is no need to expose the "trusted" aspect of the feature to users. That can be added in a more robust feature implementation later that allows users to adjust that list.

Reply:
We previously did not worry about telling the user what "Default, Default" we fellback too. I am not sure we need to now.

Reply:
I think for V1 we should only show "trusted" compilers then. Otherwise, I think users might blame our extensions for showing them an untrusted compiler. We could add untrusted compilers in a follow up PR if we can figure out a good way.